templates/node/events/package.json

```diff
@@ -13,9 +13,9 @@
     "debug": "nodemon --inspect ./node_modules/faas-js-runtime/bin/cli.js ./index.js"
   },
   "devDependencies": {
+    "cloudevents": "^3.1.0",
     "faas-js-runtime": "0.4.0",
     "nodemon": "^2.0.4",
-    "cloudevents": "^3.1.0",
```

Comment:
This change won't get picked up without running `pkger` and will conflict with https://github.com/boson-project/faas/pull/202